### ISSUE TITLE
Implement flat instead of nummod for "chapter 1", "Elizabeth II" and similar

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#558]
Precise EWT query: https://universal.grew.fr/?custom=6768e43d13bc1

**Comment by nschneid** [comment]
The above commit was done in Grew, essentially with the rule:

```grew
rule r {
  pattern { e: X-[nummod]->N; e2: X-[E:nummod]->N; X << N }
  without { X[xpos="$"] }
  without { X[lemma=foot] }
  without { N.lemma=/1982|1983|1993|200[0-5]|2008|2009/ }
  without { X.lemma=/.*(January|February|March|April|May|June|July|August|September|October|November|December).*/ }
  commands { 
    e.label = "flat";
    e2.label = "E:flat";
    X.FlatType = "Enumerated"
	}
}
```

**Comment by nschneid** [comment]
Rule for above commit:

```grew
% rehead numbered entities ("# 1", "Schedule A") analyzed as right-headed compounds so they are flat
rule r {
  pattern {
    e: X1-[compound]->X2; e2: X1-[E:compound]->X2; X2 << X1;
    X1.lemma=/[A-Z]|[0-9IVXC]+|.*[0-9].*/i;
    X2[lemma="#"|M|m|ak|EB|RER|Wii|game|party|plan|schedule|League|Strongid|BMW|paragraph|subparagraph|catch|attachment]; 
  }
  commands { 
    del_edge e; % delete the edge
    del_edge e2;
    add_edge X2 -[flat]-> X1; % add the inverse edge
    add_edge X2 -[E:flat]-> X1;
    X2.FlatType="Enumerated";
    % and then shift the in-edges pointing to X1 and redirect to X2
    shift_in X1 ==> X2;
    
    % redirect all out-edges (starting from X1) to X2 (will start from X2)
    shift_out X1 ==> X2;
  }
}
```

**Comment by nschneid** [comment]
In MISC I have specified `FlatType=Enumerated` for these. In addition, the rules detected a few cases of multipart numeric textual references like "5.1 (b)", for which I specified `FlatType=NumericLocator` (rather than attempt to choose a head based on semantics of the substructure; these are probably better tokenized as single syntactic words anyway).

**Comment by AngledLuffa** [comment]
Presumably `Rule 143` would also get this treatment?  `ISO 9002`?  Just going through ParTUT looking for similars

```
# text = The relevant standards which have been laid down in another Directive, 95/35/EC
9       in      in      ADP     E       _       11      case    _       _
10      another another DET     RI      Definite=Ind|Number=Sing|PronType=Art   11      det     _       _
11      Directive       Directive       NOUN    S       Number=Sing     7       obl     _       SpaceAfter=No
12      ,       ,       PUNCT   FF      _       7       punct   _       _
13      95      95      NUM     N       NumType=Card    17      nummod  _       SpaceAfter=No
14      /       /       PUNCT   FF      _       13      punct   _       SpaceAfter=No
15      35      35      NUM     N       NumType=Card    17      nummod  _       SpaceAfter=No
16      /       /       PUNCT   FF      _       15      punct   _       SpaceAfter=No
17      EC      EC      PROPN   SP      _       7       obl     _       SpaceAfter=No
18      ,       ,       PUNCT   FF      _       3       punct   _       _
```

```
# sent_id = en_partut-ud-247
# text = The next item is the report (A5-0104/1999)
7       (       (       PUNCT   FB      _       8       punct   _       SpaceAfter=No
8       A5-0104 A5-0104 PROPN   SP      _       6       nmod    _       SpaceAfter=No
9       /       /       PUNCT   FF      _       8       punct   _       SpaceAfter=No
10      1999    1999    NUM     N       NumType=Card    8       nummod  _       SpaceAfter=No
11      )       )       PUNCT   FB      _       8       punct   _       _
```

`Objective 1 status`, `Article 158`, and such

What about list items, `(3) The ecodesign of products...`, `1. Definitions...` etc, still `nummod`?

Reference numbers `[3]`, `as early as 1998 [1]`, still `nummod`?

Hope this isn't hijacking the thread to compare it to other treebanks

**Comment by nschneid** [comment]
Yes to the numbered rules/documents and such.

List item markers are `discourse`: https://universaldependencies.org/u/dep/discourse.html

I don't believe we have a standard for reference numbers/citations. GUM has academic texts so you could see what it uses. Feel free to open a separate issue to discuss that.

**Comment by amir-zeldes** [comment]
I'm pretty sure academic references and footnotes are still `dep` in GUM. I don't think `nummod` or `flat` are appropriate - maybe `parataxis`? But TBH they are `dep` precisely because no relation fits well...

**Comment by nschneid** [comment]
What about `discourse`? Parenthetical citation is about the structure of texts and how texts relate to one another...kind of like list item markers in a way.

**Comment by amir-zeldes** [comment]
I don't love the idea of watering down discourse more - they are really not like interjections. I suggested parataxis because they're sort of parenthetical, and really there should be `parataxis:prn` or something, but making that distinction would take a lot of manual effort.

**Comment by AngledLuffa** [comment]
I see in EWT that `day one` and `day two` now use `flat(day, one)` etc.  That looks like it is intended based on the above discussion, but, just checking before implementing it elsewhere.

`Mate 9`, a type of phone: `flat(Mate, 9)`

`Series 1`, as in `Apple Watch Series 1`, same as `Mate 9`?  `Series` head, `1` a `flat` relation

`No. 1`, as in No. 1 player in the world, `flat(No., 1)` ? seems a little weird that this isn't `nummod` considering the number is specifically putting the person in an order.  Same with `No. 31`, `No. 96`, etc

`Brisbane's Nine News`: `flat(News, Nine)`, making `News` the head?  or is `Nine` the head here?

`B-29s`... no, i refuse to tokenize this into multiple pieces anyway.  Please don't kick me out as maintainer of PUD for this insubordination... please

`Billboard 200` - `200` or `Billboard` the head?

`top 50` appears to be treated as `50` being the head and `top` an `amod`

**Comment by AngledLuffa** [comment]
also unsure: `Pluviose year III` as in the sentence 

```
In Pluviose year III (January-February 1794), easing off the pressure of the Reign of Terror, he took various measures to liberate the suspects.
```

**Comment by nschneid** [comment]
`nummod` is restricted to quantity expressions (_1 apple_, _2 apples_, _1.5 apples_). The noun+number combinations should be `flat` as described here: https://universaldependencies.org/en/dep/nmod-desc.html#numbered-entities

_Nine News_, _Billboard 200_ are headed by the 2nd word right? `compound` I guess.

_top 50_ would depend on context: "the top 50/nummod singers" vs. "singers in the top/amod 50"

**Comment by AngledLuffa** [comment]
I can do all that.  How about `Pluviose year III`, also `flat`?

**Comment by nschneid** [comment]
see my answer in #575

**Comment by nschneid** [comment]
The treatment of "aged AGE" may require a separate discussion: https://universal.grew.fr/?custom=6827f0cff1660

**Comment by amir-zeldes** [comment]
> The treatment of "aged AGE" may require a separate discussion

I feel like we've discussed this somewhere and I think I was for obl:unmarked and maybe @nschneid considered it passive? I would just do:

acl(women, aged)
obl:unmarked(aged, 15)

**Comment by AngledLuffa** [comment]
Like I did it here, then?

https://github.com/UniversalDependencies/UD_English-ParTUT/commit/eebb1fc9938bbdeabec2cb756d20321f71034c5f

**Comment by AngledLuffa** [comment]
and then I tried to do `age 49` or `age of 15` as `flat` or `nmod`:

https://github.com/UniversalDependencies/UD_English-ParTUT/commit/1063b0866f15b07bfa71dc084998f5c7df423b68

**Comment by AngledLuffa** [comment]
Everything else in PUD and ParTUT should be done, except there are some longer identifiers in ParTUT which look like this:

```
Directive 94/55/EC
report (A5-0108/1999)
Decision No 1600/2002/EC
etc etc
```

Those get tokenized, which I hate, and then often have nummod as the relation.  I suppose I can live with tokenizing them, in which case they get treated as `flat` still?  or maybe leave it untokenized and the whole thing is a single `flat` word?

**Comment by amir-zeldes** [comment]
The ages look right to me in the commits. For the internally tokenized identifiers I would almost go with goeswith or something... I would not have tokenized those apart unless there were spaces (and then I might still want to use goeswith)

**Comment by nschneid** [comment]
right-headed `compound` for "A +" and "20 something" is debatable but otherwise this is done at least for EWT
